core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueRequest.java

```diff
@@ -52,6 +55,24 @@ protected void appendToURI(StringBuilder restOfURI, String pathParam) {
         }
         restOfURI.append(pathParam);
     }
+    
+    protected void appendToURI(StringBuilder restOfURI, String queryParamName, String queryParamvalue) {
+		if (!StringUtils.endsWith(restOfURI.toString(), PATH_SEPARATOR)) {
+			if (!StringUtils.contains(restOfURI.toString(), QUERY_PARAM_NAME_VALUE_SEPERATOR)) {
```

Comment:
Can you simplify this? The if/else blocks share the last three lines, so the if/else can include the first line only.